openquake/hazardlib/tests/gsim/lanzano_2019_mod2022_test.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+#
+# Copyright (C) 2013-2020 GEM Foundation
+#
+# OpenQuake is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# OpenQuake is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with OpenQuake. If not, see <http://www.gnu.org/licenses/>.
+#
+
+from openquake.hazardlib.tests.gsim.utils import BaseGSIMTestCase
+from openquake.hazardlib.gsim.lanzano_2019 import (
+    LanzanoEtAl2019_RJB_OMO_RefRock)
+
+
+class LanzanoEtAl2019_RJB_OMO_RefRock_TestCase(BaseGSIMTestCase):
+    GSIM_CLASS = LanzanoEtAl2019_RJB_OMO_RefRock
+
+    # Tables provided by original authors
+    MEAN_FILE = "LAN2019/ITA18_MOD22_RJB_MEAN.csv"
+    STD_FILE = "LAN2019/ITA18_MOD22_RJB_STD.csv"
+
+    def test_all(self):
+        self.check(self.MEAN_FILE, self.STD_FILE,
+                   max_discrep_percentage=0.1)
+
+
```

Comment:
Why are the tests below commented out?

Reply:
I just started the tests using the ones available for the original 2019 version. For the change added this single test would be probably enough. I am waiting for feedback from the authors.